Sources/Filters/Sources/CubeSource/index.js

```diff
@@ -273,7 +273,7 @@ const DEFAULT_VALUES = {
   zLength: 1.0,
```

Comment:
line 195 should be fixed, polys should not be stored with floating number, please use Uint16Array instead (can be a new commit in the same PR)